# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/yaboot/prefix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- yaboot-1.3.10/Makefile.orig	1904-01-01 00:59:57.000000000 +0000
+++ yaboot-1.3.10/Makefile	1904-01-01 01:07:44.000000000 +0000
@@ -7,7 +7,7 @@
 DEBUG = 0
 # make install vars
 ROOT =
-PREFIX = usr/local
+PREFIX = usr
 MANDIR = man
 # command used to get root (needed for tarball creation)
 GETROOT = fakeroot
